clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -129,20 +130,27 @@ common_args = ["-key", "home_domain", args.domain,
                "-default_behaviors", "all,-bye",
                "-t", "t1"]
 
+if args.aka:
+    initial_reg_xml = "aka_initial_register.xml"
+    re_reg_xml = "aka_re_register.xml"
+else:
+    initial_reg_xml = "digest_register.xml"
+    re_reg_xml = "digest_register.xml"
+
 initial_reg_cmd = [SIPP_BIN,
-                   "-sf", SCRIPTS_DIR + "register.xml",
+                   "-sf", SCRIPTS_DIR + initial_reg_xml,
                    "-inf", REGISTER_CSV,
                    "-trace_err", "-error_file", INITIAL_REG_ERRORS,
                    "-r", str(args.initial_reg_rate),
                    "-m", str(args.subscriber_count)] + common_args + [args.icscf_target]
 
-rereg_cmd = [SIPP_BIN,
-             "-sf", SCRIPTS_DIR + "register.xml",
-             "-inf", REGISTER_CSV,
-             "-trace_err", "-error_file", RE_REG_ERRORS,
-             "-trace_stat", "-stf", RE_REG_STATS,
-             "-r", str(RPS),
-             "-m", str(REG_MAX)] + common_args + [args.icscf_target]
+re_reg_cmd = [SIPP_BIN,
```

Comment:
If we're going to change "rereg" to "re_reg" under this PR should we also do the variable [REREG_PER_HOUR](https://github.com/Metaswitch/clearwater-perf-utils/blob/f1d747d9575fc29d7c09554ee692b67dc82e1159/clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress#L71) and the [file names](https://github.com/Metaswitch/clearwater-perf-utils/blob/f1d747d9575fc29d7c09554ee692b67dc82e1159/clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress#L59) that contain rereg?